src/assets/CloudinaryFile.ts

```diff
@@ -138,6 +140,14 @@ class CloudinaryFile {
     return this;
   }
 
+  /**
+   * Sets the __cld_token__ value for the authToken signature
+   * @param token
+   */
+  setExplicitAuthToken(token: string) {
+    this.explicitAuthToken = token;
```

Comment:
I would move this to the AuthToken.
Like:
`this.AuthToken.TokenValue`  or `this.AuthToken.Token` or `this.AuthToken.Signature`

Reply:
I'm not entirely sure how I feel about this, 
Overloading the same thing can lead to confusion in runtime, but even more for the maintainers.

We can move everything to an authentication class, which can deal with everything, but we don't have that right now (And the question is do we want it?)